atd-vzd/migrations/migration_crashes_2021_01_08--1946.sql

```diff
@@ -0,0 +1,12 @@
+--
+-- Adds column for narratives that won't be automatically overwritten by crash
+-- record updates
+--
+ALTER TABLE "public"."atd_txdot_crashes" ADD COLUMN "investigator_narrative_ocr" text;
+
+UPDATE
+	atd_txdot_crashes
+SET
+	investigator_narrative_ocr = investigator_narrative
+WHERE
+	investigator_narrative IS NOT NULL
```

Comment:
This UPDATE part is a good idea, we may have to run it again after the ETL is re-enabled.